www/docs/src/content/docs/contributing/getting-started.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Getting Started
+description: Learn how to contribute to StudioCMS
+sidebar:
+  order: 1
+---
+import ContributorList from '~/components/ContributorList.astro';
+
+Thank you for investing your time in contributing to our project!
+
+Read our [Code of Conduct](https://github.com/astrolicious/studiocms?tab=coc-ov-file#code-of-conduct-) to keep our community approachable and respectable.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## Our Contributors
+
+Our project exists thanks to all the people who contribute. [Join us on GitHub](https://github.com/astrolicious/studiocms) and add your name to the list!
+
+<ContributorList />
+
+## Getting started
+
+We welcome contributions from the community! Whether it's bug reports, feature requests, or code contributions, we appreciate your help in making this project better. To navigate our codebase with confidence, see [How it works](/how-it-works/) section.
```

Comment:
This section's heading repeats the page heading. The paragraph could probably sit above "Our Contributors" I suppose?

Reply:
done in [d5e2719](https://github.com/astrolicious/studiocms/pull/372/commits/d5e2719e66d9b4f9f34208440d7f115962e5c34c)